R/bucher.R

```diff
@@ -24,9 +24,7 @@
 #'   \item{pval}{p-value of Z-test, with null hypothesis that \code{est} is zero}
 #' }
 #' @export
-#' @import stats
-#'
-#' @examples
```

Comment:
Let's leave the missing `@examples` in so we know to replace them. 
We will have trouble to get on CRAN if we don't have examples for all exported functions